codenames-front-end/src/components/game/running/CNSidePanel/CNTeamSidePanel/CNTeamSidePanel.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {CSSProperties, FC} from 'react';
+
+import css from './CNTeamSidePanel.module.css'
+import {ITeam} from "../../../../../models/CodeNames/ITeam";
+import CNTeamPlayerBlock from "../CNTeamPlayerBlock/CNTeamPlayerBlock";
+
+
+interface CNTeamSidePanelProps {
+    hidden?: boolean;
+    team: ITeam;
+    teamName: string;
+    teamColor: "blue" | "yellow";
+    position: "left" | "right";
+    className?: string;
+}
+
+const CNTeamSidePanel: FC<CNTeamSidePanelProps> = ({
+                                                       hidden = false,
+                                                       team,
+                                                       teamName,
+                                                       teamColor,
+                                                       position,
+                                                       className
+                                                   }) => {
+    return (
+        <div
```

Comment:
I see you are using A LOT of conditional styling. To handle it properly please check out https://styled-components.com/